src/Solver/time_stepping/TimeCluster.cpp

```diff
@@ -281,7 +283,7 @@ void seissol::time_stepping::TimeCluster::computeDynamicRupture( seissol::initia
   LIKWID_MARKER_STOP("computeDynamicRuptureFrictionLaw");
   }
 
-  m_loopStatistics->end(m_regionComputeDynamicRupture, layerData.getNumberOfCells(), m_globalClusterId);
+  m_loopStatistics->end(m_regionComputeDynamicRupture, layerData.getNumberOfCells(), m_profilingId);
```

Comment:
I just did a quickly read this PR which reads well (and I have no idea about the interest of the profiling data (and I don't want to hear more :-) ).
But here you change m_globalClusterId by m_profilingId = l_globalClusterId + offsetMonitoring?
if offsetMonitoring is non zero, this is not just a variable change?

Reply:
`const auto offsetMonitoring = type == Interior ? 0 : m_timeStepping.numberOfGlobalClusters;`
This allows us to have statistics for interior and copy clusters.

(One of the interesting possible results of this profiling is checking if statements such as "the cost of the dynamic rupture changes significantly over time" are true. The other, main interest for me, is creating nice plots for my thesis ;))